src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -788,39 +790,32 @@ public static IEnumerable<object[]> ToString_TestData_NotNetFramework()
             yield return new object[] { 32.5, "N100", invariantFormat, "32.5000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000" };
         }
 
-        [Fact]
-        public static void Test_ToString_NotNetFramework()
+        [Theory]
+        [MemberData(nameof(ToString_TestData_NotNetFramework))]
+        public static void Test_ToString_NotNetFramework(double d, string format, IFormatProvider provider, string expected)
```

Comment:
Corresponding tests need to be added to `Single` and potentially `Half`

Reply:
Added by [b4c6067a234b1b98b06367dfe9346b396f07bab4](https://github.com/dotnet/runtime/pull/85565/commits/b4c6067a234b1b98b06367dfe9346b396f07bab4) commit